.github/workflows/unit_tests.yml

```
@@ -0,0 +1,45 @@
+name: Unit testing with Codecov coverage checking
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  job_unit_test:
+    name: Unit test
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+      - name: Build unit tests
```

```suggestion
        uses: actions/checkout@v3
        
      - name: Build unit tests
```
Empty line between each step helps readability?

---

Done